packages/xrpl/test/client/autofill.test.ts

```diff
@@ -43,9 +46,68 @@ describe('client.autofill', function () {
 
   beforeEach(async () => {
     testContext = await setupClient()
+    amount_ = '1234'
+    paytxn = {
+      TransactionType: 'Payment',
+      Account: 'rUn84CUYbNjRoTQ6mSW7BVJPSVJNLb1QLo',
+      Amount: amount_,
+      Destination: 'rfkE1aSy9G8Upk4JssnwBxhEv5p4mn2KTy',
+      DestinationTag: 1,
+      Fee: '12',
+      Flags: 2147483648,
+      LastLedgerSequence: 65953073,
+      Sequence: 65923914,
+      SigningPubKey:
+        '02F9E33F16DF9507705EC954E3F94EB5F10D1FC4A354606DBE6297DBB1096FE654',
+      TxnSignature:
+        '3045022100E3FAE0EDEC3D6A8FF6D81BC9CF8288A61B7EEDE8071E90FF9314CB4621058D10022043545CF631706D700CEE65A1DB83EFDD185413808292D9D90F14D87D3DC2D8CB',
+      InvoiceID:
+        '6F1DFD1D0FE8A32E40E1F2C05CF1C15545BAB56B617F9C6C2D63A6B704BEF59B',
+      Paths: [
+        [{ currency: 'BTC', issuer: 'r9vbV3EHvXWjSkeQ6CAcYVPGeq7TuiXY2X' }],
+      ],
+      SendMax: '100000000',
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any -- Represent transaction JSON with `any` type
+    } as any
   })
   afterEach(async () => teardownClient(testContext))
 
+  it('Validate Payment transaction v2 API: Payment Transaction: Specify Only Amount field', async function () {
+    const txResult = await testContext.client.autofill(paytxn)
+
+    assert.strictEqual(txResult.Amount, amount_)
+  })
+
+  it('Validate Payment transaction v2 API: Payment Transaction: Specify Only DeliverMax field', async function () {
+    const payment_txn = paytxn
```

Comment:
```suggestion
    const paymentTx = paytxn
```
ditto

Reply:
updated in [c49957c](https://github.com/XRPLF/xrpl.js/pull/2689/commits/c49957c13fdc6eee0393057a662500d8af08084f)